app/assets/javascripts/mobile.js

```diff
@@ -30,10 +31,18 @@ $(document).ready(function(){
 
   /* Drawer menu */
   $('#menu_badge').bind("tap click", function(evt){
```

Comment:
I did some investigations and  my theory proved to be wrong. The fastclick library didn't change the matter. But I think I found the issue. I will comment on the relevant issue.